ilc/server/i18n.js

```diff
@@ -0,0 +1,80 @@
+const _ = require('lodash');
+const cookie = require('cookie');
+
+const DEFAULT_LOCALE = 'en-US';
```

Comment:
Perhaps, it would be nice to have this and the code below in the config
What do you think about it?